### PR TITLE
Adding meta tags

### LINKED ISSUE #9: Add favicon and app image meta tags to index head
We need the meta tags for the newly created images. These images are used for various mobile devices. We should probably add these in the Layout component file.

See this link for favicon and meta tag help:
https://github.com/audreyr/favicon-cheat-sheet

See React Helmet docs for how to use Helmet to modify the `head` element: 
https://github.com/nfl/react-helmet/tree/release/5.2.0

### PR DESCRIPTION
**Pull request by andbrl**
I have added some meta tags and changed the favicon. I am not sure if this is correct but please have a look and give me some feedback if possible. I'm also not sure if we can use bot land in the title tag? Anyways, I hope it helps with something. Cheers.

### CODE REVIEW

**Comment by cmgriffing** [comment]
Thanks.

I think we want to avoid the "Bot Land" keyword so that we don't impact any SEO.

The meta tags don't seem to include meta tags for mobile devices. It might be qworthwhile to use a generator: https://favicomatic.com/

File: src/templates/newsTemplate.js [hunk 1]
```diff
@@ -26,22 +26,22 @@ export default function Template({
...
-        path
-        title
-        featuredImage {
-          ...HeroImage
-        }
-      }
-    }
-  }
-`
+ export const pageQuery = graphql`
```
Comment (cmgriffing) [inline src/templates/newsTemplate.js]: I think the changes to this whole file were unintended.

When making commits it is a best practice to only `git add` the files you intentionally changed.

If it helps, my process is:

- `git status` to check what files have changed.
- `git diff` to see the exact changes and do an initial "code review" pass
- `git add`  with the path to the folder or files you specifically want to be part of the PR. tab completion should help

Comment (andbrl) [reply src/templates/newsTemplate.js]: Thanks. I'll get on it as soon as I get a chance!

**Comment by cmgriffing** [comment]
Not sure if you had a chance or missed the comment I left, but it would be good to undo whatever changes were done to the newsTemplate file.


I know you were just making it so it would compile.

But, remember that you can have changes in your local that you don't add to the commit.

In that case, you could have not committed it here, and created a separate PR for the compile fixes to the template.

**Comment by andbrl** [comment]
It'll probably be easier if I create a new PR.

**Pull request closed without merging**